README.md

```diff
@@ -6,52 +6,106 @@
 [![Build Status](https://github.com/gfx-rs/wgpu/workflows/CI/badge.svg)](https://github.com/gfx-rs/wgpu/actions)
 [![codecov.io](https://codecov.io/gh/gfx-rs/wgpu/branch/master/graph/badge.svg?token=84qJTesmeS)](https://codecov.io/gh/gfx-rs/wgpu)
 
-This is an implementation of [WebGPU](https://www.w3.org/community/gpu/) API in Rust, targeting both native and the Web.
-See the upstream [WebGPU specification](https://gpuweb.github.io/gpuweb/) (work in progress).
+`wgpu` is a cross-platform, safe, pure-rust graphics api. It runs natively on Vulkan, Metal, D3D12, D3D11, and OpenGLES; and on top of WebGPU on wasm.
```

Comment:
I like how you turn this around: from being a WebGPU first and native later to being just a nice native API first, that happens to be powering WebGPU implementations. 🎖️